src/com/twitter/intellij/pants/file/FileChangeTracker.java

```diff
@@ -129,7 +129,8 @@ private static ChangeType detectChangeType(Project project, @NotNull VirtualFile
     if (rootManager.getFileIndex().getContentRootForFile(file) != null) {
       boolean isBuildFile = PantsUtil.isBUILDFileName(file.getName());
       boolean isUnderPantsRepo = PantsUtil.isFileUnderPantsRepo(file);
-      if (isUnderPantsRepo && isBuildFile) {
+      boolean isNotInPantsD = !file.getPath().contains("/.pants.d/");
```

Comment:
Can we somehow use the `pants_workdir` from PantsOptions, instead of harcoding `/.pants.d/` because the workdir (default '.pants.d/') can actually be configured?

nit: can we use `isInPantsD` then use `!isInPantsD` in the if statement.

Reply:
Fixed - I use `pants_workdir` as the default and checking for `/.pants.d/` as a fallback. It seems though that `pants_workdir` is always defined, so the fallback might not be actually needed.